lighthouse-core/config/default-config.js

```diff
@@ -31,6 +31,12 @@ const UIStrings = {
   diagnosticsGroupTitle: 'Diagnostics',
   /** Description of the diagnostics section of the Performance category. Within this section are audits with non-imperative titles that provide more detail on the page's page load performance characteristics. Whereas the 'Opportunities' suggest an action along with expected time savings, diagnostics do not. Within this section, the user may read the details and deduce additional actions they could take. */
   diagnosticsGroupDescription: 'More information about the performance of your application.',
+  /** Title of the Accessibility category of audits. Accessibility contains audits focused on making web content accessible to users with disabilities. Also used as a label of a score gauge; try to limit to 20 characters. */
+  a11yCategoryTitle: 'Accessibility',
+  /** Description of the Accessibility category. This is displayed immediately under the Accessibility title element. No character length limits. 'improve the accessibility of your web app' becomes link text to additional documentation. */
+  a11yCategoryDescription: 'These checks highlight opportunities to [improve the accessibility of your web app](https://developers.google.com/web/fundamentals/accessibility). Only a subset of accessibility issues can be automatically detected so manual testing is also encouraged.',
+  /** Description of the Accessibility manual checks category. This is displayed above the manual checks section of the Accessibility category. No character length limits. 'conducting an accessibility review' becomes link text to additional documentation. */
```

Comment:
the second sentence doesn't clarify much. Maybe something like "This description is displayed above a list of accessibility audits that currently have no automated test and so must be verified manually by the user. No character length limits...".